lib/src/internal/utils/utils.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:momento/momento.dart';
+import 'package:momento/src/errors/errors.dart';
+
+Value getStringOrBinaryFromDynamic(dynamic value, String parameterName) {
```

Comment:
this is a really cool thing to try!

I'm guessing the editor isn't able to give any type hints for this at all though? and it would be a runtime error if you just tried to pass in like, some random object type as the argument to one of these dynamic args? If so, lemme noodle on that.

Reply:
If type hints are a good trade for an uglier API, we could add the type to the function name like we do with some of the collection methods in Java: `dictionarySetFieldsStringBytes`.